GCSViews/ConfigurationView/ConfigRawParams.cs

```diff
@@ -318,11 +359,14 @@ private void BUT_writePIDS_Click(object sender, EventArgs e)
                     CustomMessageBox.Show("Set " + value + " Failed");
                 }
             }
+            
 
             if (error > 0)
                 CustomMessageBox.Show("Not all parameters successfully saved.", "Saved");
+            else if (temp.Count>0)
+                CustomMessageBox.Show($"{temp.Count} parameters successfully saved.", "Saved");
             else
-                CustomMessageBox.Show("Parameters successfully saved.", "Saved");
+                CustomMessageBox.Show($"No parameters were changed.", "No changes");

```

Comment:
Minor nitpick: you don't need the `$` on this one.

Reply:
removed in [commit id 22c75ed](https://github.com/ArduPilot/MissionPlanner/pull/3412/commits/22c75ed012c5702bd9eb70df164d8166320fa3e3)